examples/seismic/abc_methods/01_introduction.ipynb

```diff
@@ -0,0 +1,1187 @@
+{
```

Comment:
<p>Overall I think you can replace the plotting routines you have here with what is used in devito.</p><p>PLotting fields, velocities, shotrecords from receivers etc, has been done already.</p><p>If you look here : /examples/seismic/tutorials/03_fwi.ipynb</p><p>This example contains I think all the plotting cases.</p><p>Again everything is going in the correct direction. All info is very detailed. Thanks for the work you are putting into this.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1323/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
I know that there are functions to plot the values of receivers and displacament, but these functions using the model structure and in these notebooks we don't use these structure.